test_scripts/Defects/5_0/1919_Multiple_media_and_navigation_apps_activation_during_active_embedded_audio_source_audio_mixing_supported.lua

```diff
@@ -0,0 +1,137 @@
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == false and
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "DEFAULT"
+      then
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated", { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true
+      then
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.videoStreamingState == "STREAMABLE" and
+          data.payload.hmiLevel == "LIMITED" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.videoStreamingState == "NOT_STREAMABLE" and
+          data.payload.hmiLevel == "BACKGROUND" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function embededAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "EMBEDDED_NAVI", isActive = true })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus", {
+    hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE"
+  })
+end
+
+local function activateMediaApp()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "EMBEDDED_NAVI", isActive = false })
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus",{ hmiLevel = "FULL", audioStreamingState = "AUDIBLE" })
+end
+
+local function activateNaviApp()
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(2) })
+  EXPECT_HMIRESPONSE(requestId)
+  common.getMobileSession(2):ExpectNotification("OnHMIStatus",
+  { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" },
+  { hmiLevel = "FULL", audioStreamingState = "AUDIBLE" })
+  :Times(2)
+
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus", { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE" })
+end
+
+local function activateNonMediaApp()
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(3) })
+  EXPECT_HMIRESPONSE(requestId)
+  common.getMobileSession(3):ExpectNotification("OnHMIStatus", {hmiLevel = "FULL", audioStreamingState = "NOT_AUDIBLE" })
+end
+
+--[[ Test ]]
+runner.Title("Preconditions")
```

Comment:
"MixingAudioSupported" should be set to true in .ini file

Reply:
@GetmanetsIrina You can find the updates in the commit cd2fa7e